modules/virt-monitoring-upgrade-status.adoc

```diff
@@ -4,9 +4,9 @@
 
 :_mod-docs-content-type: PROCEDURE
 [id="virt-monitoring-upgrade-status_{context}"]
-= Monitoring {VirtProductName} upgrade status
+= Monitoring update status
```

Comment:
Should it be `upgrade`?

Reply:
> Use "update" when referring to updating the cluster to a new version. Although "upgrade" is sometimes used interchangeably, "update" is the preferred term to use, for consistency.

I had switched this to "update" for consistency with this OpenShift guidance ^